bento_wes/runs.py

```diff
@@ -288,6 +291,57 @@ def _create_run(db, c):
         return flask_bad_request_error("Assertion error: bad run request format")
 
 
+def fetch_run_details(c, public_endpoint=False):
+    c.execute(
+        "SELECT r.id AS run_id, r.state AS state, rr.*, rl.* "
+        "FROM runs AS r, run_requests AS rr, run_logs AS rl "
+        "WHERE r.request = rr.id AND r.run_log = rl.id"
+    )
+
+    runs = []
+    for r in c.fetchall():
+        request = run_request_dict(r) if not public_endpoint else run_request_dict_public(r)
+        run_log = run_log_dict(r["run_id"], r) if not public_endpoint else run_log_dict_public(r["run_id"], r)
+        end_time = run_log['end_time']
+        state = r['state']
```

Comment:
use double quotes for all strings unless nested in an f-string for consistency with the rest of the codebase